lte/gateway/configs/templates/mme.conf.template.bak

```diff
@@ -0,0 +1,325 @@
+################################################################################
```

Comment:
is this added to PR by accident?

Reply:
Yes, removed. Thank you for pointing it out